ccd-adapter/src/main/java/uk/gov/hmcts/cmc/ccd/mapper/EvidenceMapper.java

```diff
@@ -22,7 +22,7 @@ public EvidenceMapper(EvidenceRowMapper evidenceRowMapper) {
 
     @Override
     public CCDEvidence to(Evidence evidence) {
-        if (evidence == null) {
+        if (evidence == null || evidence.getRows() == null || evidence.getRows().isEmpty()) {
```

Comment:
why not use something like CollectionUtils.isEmpty(evidence.getRows()) which will check both conditions